### DISCUSSION TITLE
How to make annotation hotspots in different languages?

### DISCUSSION THREAD

**Discussion by wawa2016** [google/model-viewer#1845] [Q&A] [answered]
I would like to have my annotation hotspots in two languages. Could you please advice on how to do it?
Ideally I would have a UI button that allows me to toggle between two sets of hotspots that I translate.
Also could you give me a hint how can I create a toggle button that controls the visibility of the hotspots please?
I appreciate your input. Please take into account that I am new to coding :)

**Comment by elalish** [comment, marked as answer]
Since hotspots are just standard HTML, you'll want to look into how this is done for any website. `display:none` is popular for hiding elements and I believe there are libraries to aid with localization. Anything you find should be applicable directly to hotspots as well.
